src/types/alerts.ts

```diff
@@ -0,0 +1,19 @@
+export namespace Alerts {
+  export interface AlertConfig {
+    slack?: SlackAlert;
+    email?: EmailAlert;
+    url?: WebhookAlert;
+  }
+
+  export interface SlackAlert {
```

Comment:
Same for this and email/webhook, you can call this `Slack` and reference it as `Alerts.Slack`.